web/react/utils/utils.jsx

```diff
@@ -201,40 +203,17 @@ export function displayDate(ticks) {
     return monthNames[d.getMonth()] + ' ' + d.getDate() + ', ' + d.getFullYear();
 }
 
-export function displayTime(ticks, utc) {
-    const d = new Date(ticks);
-    let hours;
-    let minutes;
-    let ampm = '';
-    let timezone = '';
-
-    if (utc) {
-        hours = d.getUTCHours();
-        minutes = d.getUTCMinutes();
-        timezone = ' UTC';
-    } else {
-        hours = d.getHours();
-        minutes = d.getMinutes();
-    }
-
-    if (minutes <= 9) {
-        minutes = '0' + minutes;
-    }
-
+export function displayTime(ticks) {
     const useMilitaryTime = PreferenceStore.getBool(Constants.Preferences.CATEGORY_DISPLAY_SETTINGS, 'use_military_time');
-    if (!useMilitaryTime) {
-        ampm = ' AM';
-        if (hours >= 12) {
-            ampm = ' PM';
-        }
-
-        hours = hours % 12;
-        if (!hours) {
-            hours = '12';
-        }
-    }
 
-    return hours + ':' + minutes + ampm + timezone;
+    return (
+        <FormattedTime
+            value={ticks}
+            hour='numeric'
+            minute='numeric'
+            hour12={!useMilitaryTime}
+        />
+    );
 }
 
 export function displayDateTime(ticks) {
```

Comment:
I was really confused for a bit as I grep'd my local copy again looking for other uses of `Utils.displayTime` and couldn't find any (there weren't any when I originally made my changes), but it appears @jwilander's PR that added it's use was merged an hour after I submitted this PR.  Good catch, thanks @coreyhulen, I'll change this tomorrow when I'm in.